scss/lessonCard.scss

```diff
@@ -0,0 +1,43 @@
+* {
+  color: #292929;
```

Comment:
`*` selectors shouldn't be applied here, it should be in a file that feels more global.

Reply:
We don't want to use `*` selector. Since, `scss` is not scoped, it's changing in colors in places we don't want. 

For example, the text inside the dark purple section in the Landing Page is `#292929` instead of `white`